identity-service/src/routes/cognito.js

```diff
@@ -75,31 +75,58 @@ module.exports = function (app) {
           headers
         })
 
+        // https://cognitohq.com/docs/reference#flow_get_flow_session
+        // user is always present
         const userInfo = flowSessionResponse.data.user
+
+        // id_number is always present but nullable
+        // phone is always present but nullable
+        // date_of_birth is always present but nullable
+        // address is always present but nullable
+        // name is always present but nullable
```

Comment:
no need to do it now, but we should soon port this file to typescript and just put this in the type system :)

Reply:
for sure